server/spec/utils.js

```diff
@@ -182,16 +197,14 @@ exports.expectTouchTimestamps = function(record, options) {
     throw new Error('Record must have an updated_at property!');
   }
 
-  options = options || { hasBeenCreated: true }
-
   expect(created).to.be.an.instanceOf(Date);
   expect(updated).to.be.an.instanceOf(Date);
-  if (options.hasBeenCreated) {
+  if (_.get(options, 'hasBeenCreated', true)) {
```

Comment:
Why is this necesseray ?
Doesn't `if(options.hasBeenCreated)` will turn out `false` if the property doesn't exist ?

Reply:
It replaces the previous `options = options || { hasBeenCreated: true }` that was a few lines above. Lodash's `get` returns the specified default value (true in this case) if the object's property resolves to undefined (which will be the case if it's not set or set to undefined).